packages/admin/admin/src/form/FinalFormInput.tsx

```diff
@@ -23,28 +24,51 @@ export function FinalFormInput({
     ...props
 }: FinalFormInputProps): React.ReactElement {
     const type = props.type ?? input.type ?? "text";
-    const { enabled: translationEnabled, translate } = useContentTranslationService();
+    const { enabled: translationEnabled, showApplyTranslationDialog, translate } = useContentTranslationService();
     const isTranslatable = translationEnabled && !disableContentTranslation && type === "text" && !props.disabled;
 
+    const [pendingTranslation, setPendingTranslation] = React.useState<boolean>(false);
+    const [translation, setTranslation] = React.useState<string | undefined>(undefined);
```

Comment:
Also, this should be pendingTranslation

Reply:
c3b6cf25126d9d56fcf9186a3f3057dd0ee86ffa